gotfparse/pkg/converter/hacks.go

```diff
@@ -0,0 +1,13 @@
+package converter
+
+import (
+	"reflect"
+	"unsafe"
+)
+
+func getPrivateValue(obj interface{}, key string) reflect.Value {
```

Comment:
Can you add a docstring for why this is necessary?

Reply:
I added some text above where it's used, does that work?